uaclient/files/files.py

```diff
@@ -35,7 +35,10 @@ def is_private(self) -> bool:
     def is_present(self):
         return os.path.exists(self.path)
 
-    def write(self, content: str):
+    def write(self, content: Any):
+        self._write(str(content))
```

Comment:
Using `Any` here removes all the help from mypy.

I'd rather have different classes that use different types than use `Any`. For example, rather than `ProJSONFile` _extending_ `UAFile`, it could _have_ a `UAFile` passed to the constructor.

Reply:
Or, it could extend and overwrite the method :D